packages/components/forms/src/Multiselect/Multiselect.styles.ts

```diff
@@ -0,0 +1,74 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+export const getMultiselectStyles = () => ({
+  multiselect: css({
+    position: 'relative',
+    width: '100%',
+  }),
+  currentSelection: css({
+    width: '50%',
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    verticalAlign: 'bottom',
+    marginRight: tokens.spacing2Xs,
+  }),
+  inputField: css({
+    paddingRight: tokens.spacingXl,
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  }),
+  toggleButton: css({
+    position: 'absolute',
+    top: '1px',
+    right: '1px',
+    zIndex: 1,
+    padding: tokens.spacing2Xs,
+    height: '38px',
+  }),
+  content: (listMaxHeight: number) =>
+    css({
+      overflow: 'auto',
+      maxHeight: `${listMaxHeight}px`,
+    }),
+  list: css({
+    listStyle: 'none',
+    padding: `${tokens.spacingXs} 0`,
+    margin: 0,
+  }),
+  groupTitle: css({
+    padding: `${tokens.spacingXs} ${tokens.spacingM}`,
+    lineHeight: tokens.lineHeightM,
+  }),
+  noMatchesTitle: css({
+    color: tokens.gray500,
+    margin: tokens.spacingM,
+  }),
+  item: (isDisabled?: boolean) =>
+    css({
+      padding: `${tokens.spacingXs} ${tokens.spacingM}`,
+      wordBreak: 'break-word',
+      whiteSpace: 'break-spaces',
+      cursor: isDisabled ? 'not-allowed' : 'pointer',
+      hyphens: 'auto',
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'center',
+      opacity: isDisabled ? 0.5 : 1,
+
+      '&:focus, &:hover': {
+        backgroundColor: tokens.gray100,
+      },
+      '&:active': {
+        backgroundColor: tokens.gray200,
+      },
```

Comment:
Can we use the keyboard-only focus styling here? See the Menu component for example.

Reply:
do you mean the ':focus-visible' style? not sure what you mean

Reply:
Sorry, to be clear it's that selector yes, but we have to apply it in an annoying order to avoid breaking focus state for for browsers who don't implement it yet. Looks like this: https://github.com/contentful/forma-36/blob/a64a2ce00e1ae300228ff87d68d1fdebc61787e2/packages/components/button/src/Button/Button.styles.ts#L59-L67